.github/dependabot.yml

```diff
@@ -5,3 +5,8 @@ updates:
     directory: "/"
     schedule:
       interval: "monthly"
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Could we do this monthly instead? Xcaddy doesn't change much...

Reply:
Updated